README.md

```diff
@@ -100,6 +100,7 @@ shell command: `printf "0x%04x%04x\n" $RANDOM $RANDOM`
 * STM32F1 - 0x5ee21072
 * STM32F4 - 0x57755a57
 * ATmega32 - 0x16573617
+* FX2 - 0x5a18069b
```

Comment:
```suggestion
* Cypress FX2 - 0x5a18069b
```

Great! Would be nice to have a link to your repo, if it's public.

Reply:
Should we also add vendor names for all the other chip series? (I would personally prefer to see it)

Repo wasn't public at the time when I sent this PR, but it is now: https://github.com/whitequark/libfx2

Reply:
Sure, though for STM32 it would be a bit redundant ;)